linera-views/src/dynamo_db.rs

```diff
@@ -539,6 +538,7 @@ impl DynamoDbStoreInternal {
             .expression_attribute_values(":prefix", AttributeValue::B(Blob::new(key_prefix)))
             .set_exclusive_start_key(start_key_map)
             .send()
+            .boxed()
```

Comment:
Is the boxed work independent on the work on LocalKeyValueStore?

Reply:
You're right, I've moved it to https://github.com/linera-io/linera-protocol/pull/1860 :)